manifests/controller-manager/vsphere-cloud-controller-manager-ds.yaml

```diff
@@ -37,6 +37,9 @@ spec:
         effect: NoSchedule
       - key: node-role.kubernetes.io/master
         effect: NoSchedule
```

Comment:
The default value for operator is Equal. It seems weird to have node-role.kubernetes.io/master with an operator of both exists and equal. Given that no value was provided the equals seems wrong anyway, I assume you should have just exists. Tagging in @andrewsykim as he knows this cloud provider better than I do.

Reply:
Do you mean remove the old one and keep my version?

Reply:
Yes.
I don't believe it makes sense to have both. Exists says the mere presence of the key is sufficient and the value is irrelevant. Equal says the key must exist and the value must match the given key. Having both seems wrong. At best its confusing and at worst it will trigger some odd intermittent ordering bug. Assuming exists is what you want, then you should remove the corresponding equal.

Reply:
Gotcha; updated the PR 👍